### PR TITLE
New documentation structure

### LINKED ISSUE #317: improve documentation
- link https://docs.exasol.com/database_concepts/virtual_schemas.htm maybe reduce redundancy; right now it is unclear which tutorial to use. I used both. Gihub and the documentation.
- add notice like: "Unlike the External Table of other databases, Exasols Virtuals Schemas do not only link one table but a whole schema. This makes setup much easier as not each individual table needs to be mapped. If only one table shall be mapped, this can be achieved by using TABLE_FILTER. The schema of the external table is provided by the Virtual Schema on `CREATE VIRTUAL SCHEMA` or on REFRESH..."
- add notice on predefine schema like: "The schema (table structure) of the Virtual Schema is stored in the Exasol database. That implies that only coulmns defined in the schema can be part of the request." (if so)

### PR DESCRIPTION
**Pull request by jakobbraun**
fixes #317

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,13 @@ Regardless of whether that source is a relational database like ours, or it's li
 
 This means an user familiar with SQL will immediately feel at home when accessing remote data through Virtual Schemas.
 
+Other databases vendors offer similar features and refer to them as `External Tables` or `Foreign Data Wrapper (FDW)`.
```
Comment (AnastasiiaSergienko) [inline README.md:29]:
```suggestion
Virtual Schemas can be also described in known terms as `External Tables` or `Foreign Data Wrapper (FDW)`.
```

Comment (AnastasiiaSergienko) [reply README.md:29]: Or something like this

File: README.md [hunk 2]
```diff
@@ -26,6 +26,13 @@ Regardless of whether that source is a relational database like ours, or it's li
...
+
+This repository contains Virtual Schema adapters for different external data sources.
+Each of these adapters translates between the Exasol SQL dialect and the SQL dialect of the external data source.
```
Comment (AnastasiiaSergienko) [inline README.md:32]: These two sentences look redundant to me.  The line 23 has the same information

File: README.md [hunk 3]
```diff
@@ -26,6 +26,13 @@ Regardless of whether that source is a relational database like ours, or it's li
...
+
+
+
```
Comment (AnastasiiaSergienko) [inline README.md:35]: Leave only one empty line between text and the header

File: README.md [hunk 4]
```diff
@@ -74,6 +81,12 @@ in all your `CREATE JAVA ADAPTER SCRIPT` statements. The old entry point was rem
 * Allows redirecting log output to a remote machine
 * Allows remote debugging with the Java Debugger
 
+## Limitations
+
+* Supports only read access to the external data
+* The Virtual Schema adapter takes about 1s for startup.
```
Comment (AnastasiiaSergienko) [inline README.md:87]:
```suggestion
* The Virtual Schema adapter takes about 1s to start.
```

File: README.md [hunk 5]
```diff
@@ -82,10 +95,12 @@ This is an open source project which is officially supported by Exasol. For any
...
 ## Information for Users
 
-Virtual Schemas support 16 different SQL dialects, from OpenSource databases like PostgreSQL to commercial products like Oracle. 
-Please refer to the user-guide for a [full list of supported SQL dialects](doc/user-guide/user_guide.md#list-of-supported-dialects)
+* [Getting started with Virtual Schemas](https://docs.exasol.com/database_concepts/virtual_schemas.htm)
+* [List of Supported Dialect](doc/user-guide/dialects.md)
 
-* [User Guide](doc/user-guide/user_guide.md)
+Additional resources:
+* [Adapter Properties Reference](doc/user-guide/adapter_properties.md)
```
Comment (AnastasiiaSergienko) [inline README.md:102]: Let's rename it to `Virtual Schema's Properties` and rename the file itself too. Adapter sounds too misleading

File: doc/user-guide/add_to_user_guide.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+# Unrecognized and Unsupported Data Types
```
Comment (AnastasiiaSergienko) [inline doc/user-guide/add_to_user_guide.md:1]: First - the file's name doesn't tell me what's inside. Second - I can't see any link to this file, nobody will ever reach it without a link...

File: doc/user-guide/user_guide.md [hunk 7]
```diff
@@ -1,238 +1,2 @@
...
-[sql-server-dialect-doc]:../dialects/sql_server.md
-[sybase-dialect-doc]: ../dialects/sybase.md
-[teradata-dialect-doc]: ../dialects/teradata.md
-
-[virtual-schemas-repository]: https://github.com/exasol/virtual-schemas
-[virtual-schemas-releases]: https://github.com/exasol/virtual-schemas/releases
-[exasol-virtual-schema-repository]: https://github.com/exasol/exasol-virtual-schema
-[exasol-virtual-schema-releases]: https://github.com/exasol/exasol-virtual-schema/releases
\ No newline at end of file
+* [Getting started with Virtual Schemas](https://docs.exasol.com/database_concepts/virtual_schemas.htm)
```
Comment (AnastasiiaSergienko) [inline doc/user-guide/user_guide.md:1]: The existence of this file doesn't make sense anymore.

File: README.md (hunk 2 above)
Comment (jakobbraun) [reply README.md:32]: I tried to introduce the ther `Virtual Schema adapter` here. To make a difference between `Virtual schema `and `Virtual Schema adapter`

File: doc/user-guide/user_guide.md [hunk 8]
```diff
@@ -1,71 +1,41 @@
-# Supported Dialects
+#User Guide
```
Comment (AnastasiiaSergienko) [inline doc/user-guide/user_guide.md:1]: Link to this user guide from README. You can remove the link later when the documentation is updated

File: doc/user-guide/user_guide.md [hunk 9]
```diff
@@ -1,71 +1,41 @@
...
-# JDBC Adapter for Virtual Schemas
-
-## Overview
 
 The JDBC adapter for virtual schemas allows you to connect to JDBC data sources like Hive, Oracle, Teradata, Exasol or any other data source supporting JDBC.
 It uses the well proven ```IMPORT FROM JDBC``` Exasol statement behind the scenes to obtain the requested data, when running a query on a virtual table. 
 The JDBC adapter also serves as the reference adapter for the Exasol virtual schema framework.
 
-You can find supported dialects in [the list of supported dialects](#list-of-supported-dialects).
+You can find supported dialects in [the list of supported dialects](dialects.md).
```
Comment (AnastasiiaSergienko) [inline doc/user-guide/user_guide.md:8]: Make sure that this link works. My browser shows it as a broken link. Check also other linked you on the page and also links to the sections you removed in other parts of documentation

File: doc/user-guide/user_guide.md [hunk 10]
```diff
@@ -1,71 +1,41 @@
...
 
 ## Getting Started
 
-This page contains common information applicable to all the dialects. You can also check more [detailed guides for each dialect](#list-of-supported-dialects) in the dialects' documentation.
+This page contains common information applicable to all the dialects. You can also check more [detailed guides for each dialect](dialects.md) in the dialects' documentation.
+
+The steps for creating virtual schema are:
+
+* Deploying JDBC Driver Files
+* Install the adapter script
```
Comment (AnastasiiaSergienko) [inline doc/user-guide/user_guide.md:27]: Use nouns everywhere or rephrase the sentence and use verbs everywhere. It's a mix with strange grammar now

File: doc/user-guide/user_guide.md [hunk 11]
```diff
@@ -74,7 +44,7 @@ You have to upload the JDBC driver files of your remote database **twice** (exce
 Note that some JDBC drivers consist of several files and that you have to upload all of them. 
 To find out which JAR you need, check the individual dialects' documentation pages.
 
-## Deploying the Adapter Script
+## Install the adapter script
```
Comment (AnastasiiaSergienko) [inline doc/user-guide/user_guide.md:47]: Let's also unify headers. Nouns or verbs should be used consistently

File: doc/user-guide/user_guide.md [hunk 12]
```diff
@@ -92,25 +62,31 @@ CREATE JAVA ADAPTER SCRIPT SCHEMA_FOR_VS_SCRIPT.JDBC_ADAPTER_SCRIPT AS
...
 
-The following statements demonstrate the common steps to create a Virtual Schema. 
-You can find the similar tutorial with dialect-specific information for each dialect in the [supported dialects list](#supported-dialects). 
+## Define a named connection.
 
-Please scroll down to see a list of all properties supported by the JDBC adapter.
-
-First we create a connection and then a virtual schema. The adapter will retrieve the metadata via JDBC and map them to virtual tables. 
-The metadata (virtual tables, columns and data types) are then cached in Exasol.
+First we create a connection and then a virtual schema.
```
Comment (AnastasiiaSergienko) [inline doc/user-guide/user_guide.md:68]: Remove here a virtual schema mentioning as you divided them into two sections

File: doc/user-guide/user_guide.md (hunk 8 above)
Comment (AnastasiiaSergienko) [inline doc/user-guide/user_guide.md:1]:
```suggestion
# User Guide
```

**Comment by AnastasiiaSergienko** [comment]
Please, also update changes file: add your ticket's numbers and comments what was changed there

File: README.md [hunk 13]
```diff
@@ -26,6 +26,11 @@ Regardless of whether that source is a relational database like ours, or it's li
 
 This means an user familiar with SQL will immediately feel at home when accessing remote data through Virtual Schemas.
 
+Virtual Schemas can be also described in known terms as `External Tables` or `Foreign Data Wrapper (FDW)`.
+
+This repository contains Virtual Schema adapters for different external data sources.
+Each of these adapters translates between the Exasol SQL dialect and the SQL dialect of the external data source.
```
Comment (jakobbraun) [inline README.md:32]:
```suggestion
```

File: doc/user-guide/user_guide.md [hunk 14]
```diff
@@ -1,71 +1,41 @@
...
+See [SQL Client Specifics](sql_clients.md) for details
 
 ## Getting Started
 
-This page contains common information applicable to all the dialects. You can also check more [detailed guides for each dialect](#list-of-supported-dialects) in the dialects' documentation.
+This page contains common information applicable to all the dialects. You can also check more [detailed guides for each dialect](dialects.md) in the dialects' documentation.
+
+The steps for creating virtual schema are:
+
+* Deploying JDBC Driver Files
```
Comment (jakobbraun) [inline doc/user-guide/user_guide.md:26]:
```suggestion
* Deploy JDBC Driver Files
```

File: doc/user-guide/user_guide.md [hunk 15]
```diff
@@ -1,71 +1,41 @@
...
+* Install the adapter script
+* Define a named connection
+* Create Virtual Schema
 
-Before you can start using the JDBC adapter for virtual schemas you have to deploy the adapter and the JDBC driver of your data source in your Exasol database.
 
 ## Deploying JDBC Driver Files
```
Comment (jakobbraun) [inline doc/user-guide/user_guide.md:32]:
```suggestion
## Deploy JDBC Driver Files
```

File: doc/user-guide/user_guide.md (hunk 9 above)
Comment (jakobbraun) [reply doc/user-guide/user_guide.md:8]: I just checkt it agian. For me it works.

File: doc/user-guide/user_guide.md (hunk 12 above)
Comment (jakobbraun) [reply doc/user-guide/user_guide.md:68]:
```suggestion
First we create a connection.
```

File: doc/user-guide/user_guide.md [hunk 16]
```diff
@@ -92,25 +62,31 @@ CREATE JAVA ADAPTER SCRIPT SCHEMA_FOR_VS_SCRIPT.JDBC_ADAPTER_SCRIPT AS
...
-
-First we create a connection and then a virtual schema. The adapter will retrieve the metadata via JDBC and map them to virtual tables. 
-The metadata (virtual tables, columns and data types) are then cached in Exasol.
+First we create a connection and then a virtual schema.
 
 ```sql
 CREATE CONNECTION JDBC_CONNECTION TO '<jdbc connection string>' USER 'usr' IDENTIFIED BY 'pwd';
+```
+
+## Create Virtual Schema
```
Comment (jakobbraun) [inline doc/user-guide/user_guide.md:75]:
```suggestion

Now we create the Virtual Schema adapter script.
```

**Pull request merged**